src/handlers/handler.ts

```diff
@@ -0,0 +1,7 @@
+import { Context } from "../types/context";
+
+export async function doSomething(context: Context) {
```

Comment:
what is this for?

Reply:
to do something

nah it's just a placeholder and is called in `plugin.ts`, I can remove if you'd prefer

Reply:
removed in favour of a comment